<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmHora</class>
 <widget class="QDialog" name="FrmHora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horas del Evento</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 48, 58);</string>
  </property>
  <widget class="QPushButton" name="BtnTodos">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
   </property>
   <property name="text">
    <string>Todas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnNinguna">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
   </property>
   <property name="text">
    <string>Ninguna</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancelar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAceptar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>171</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="H0">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="H1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="H2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="H3">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="H4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="H5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="H6">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="H7">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="H8">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="H9">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="H10">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="H11">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="H12">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="H13">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="H14">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="H15">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="H16">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="H17">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="H18">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="H19">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="H20">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="H21">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>21</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="H22">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>22</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="H23">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(32, 39, 47);
   border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string>23</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>BtnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>FrmHora</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
